src/timetell/datapunt.sql

```diff
@@ -1,495 +1,3 @@
-
-CREATE MATERIALIZED VIEW "{schemaname}"."v_timetell_projectenoverzicht_3" AS
```

Comment:
In het bestand `timetell/importer.py` staan op de regels 950, 951 en 952 (https://github.com/Amsterdam/timetell/blob/master/src/timetell/importer.py#L950) de views die we aan het einde van de import controleren. Als deze niet bestaan, dan gaat de import alsnog fout. 

Omdat we de views nu verwijderen, moeten ze daar ook weg.